tools/interactive/interactivetool_cellxgene.xml

```diff
@@ -1,6 +1,6 @@
-<tool id="interactive_tool_cellxgene" tool_type="interactive" name="Interactive CellXgene Environment" version="0.16.2">
+<tool id="interactive_tool_cellxgene" tool_type="interactive" name="Interactive CellXgene Environment" version="1.1.1">
```

Comment:
I'd generally advocate for adding new versions in a separate file, so users can switch.

Reply:
Do I need to edit any other file for these or will these be picked up because they are in the specific folder? Sorry, I'm spoiled by years of managing tools with the toolshed, so I'm not sure how to manually deal with multiple versions of tools.